### DISCUSSION TITLE
Duplicate subscription plan ids

### DISCUSSION THREAD

**Discussion by kaijuliu-chewielabs** [swellstores/community#437] [Help] [answered]
It's possible to have products with duplicate subscription plan ids. I believe this occurs when you add a subscription plan to a product with existing plans and then drag the new plan into an existing index. I haven't tried to repro this directly, but we have evidence of this across all of our stores. This is fairly disastrous from a programmatic perspective because we specify subscription plans by id in our orders.

**Comment by logeshswell** [comment, marked as answer]
Yes, you're right, it's a bug. Let me file a bug ticket and keep you posted when I get some updates.

**Comment by steven-k1** [comment]
Hello @kaijuliu-chewielabs A fix was released for this. If you continue to see any issues, let us know.
